charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -156,6 +157,44 @@ spec:
                         serviceAccountKeyJSON:
                           type: string
                           minLength: 1
+                    s3Compatible:
+                      type: object
+                      required:
+                        - bucket
+                        - endpoint
+                      oneOf:
+                      - required:
+                        - accessKeyID
+                        - secretAccessKey
+                        allOf:
+                        - not:
+                            required:
+                            - secretName
+                      - required:
+                        - secretName
+                        allOf:
+                        - not:
+                            required:
+                            - accessKeyID
+                            - secretAccessKey
+                      properties:
```

Comment:
style nit: move the `s3Compatible.properties` section right below the `s3Compatible.required` section.